Tools/scripts/board_list.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+'''
+list of boards for build_binaries.py and custom build server
+'''
+
+AUTOBUILD_BOARDS = ["fmuv2",
```

Comment:
This should probably have been a method returning this list as we did want to NOT have a static list of boards in this directory.  Still, easy enough to fix later.